everviz/plugins/objectives_plot/objectives_plot.py

```diff
@@ -247,26 +248,44 @@ def update_graph(
                         ]
                     )
                 else:
-                    y_values = func_data["value"]
-                    if radio_value == "Normalized":
-                        y_values = y_values * func_data["norm"]
-                    elif radio_value == "Weighted + Normalized":
-                        y_values = y_values * func_data["norm"] * func_data["weight"]
 
-                    traces.append(
-                        go.Scatter(
-                            y=y_values,
-                            x=func_data["batch"],
-                            mode=plot_mode,
-                            marker={"size": 10},
-                            name=key,
-                            showlegend=True,
+                    realizations = func_data["realization"].unique()
+
+                    for real in realizations:
+                        line = func_data[func_data["realization"].isin([real])]
+                        y_values = line["value"]
+                        if radio_value == "Normalized":
+                            y_values = y_values * line["norm"]
+                        elif radio_value == "Weighted + Normalized":
+                            y_values = y_values * line["norm"] * line["weight"]
+                        line_style = {"dash": "dash"}
+                        if len(func_list) > 1:
+                            line_style["color"] = color
+
+                        if hoverData is not None:
+                            hover_real = hoverData["points"][0]["customdata"]
```

Comment:
This results in an error when the user hovers away from a point 
```
/everest/lib/python3.6/site-packages/everviz/plugins/objectives_plot/objectives_plot.py", line 266, in update_graph
    hover_real = hoverData["points"][0]["customdata"]
KeyError: 'customdata'
```
I suggest using 

`hover_real = hoverData["points"][0].get("customdata", None)`

or better yet
`clicked_realization = click_data["points"][0].get("customdata", None)`